src/clj_kondo/impl/analyzer/namespace.clj

```diff
@@ -41,19 +41,34 @@
         children (:children libspec-expr)
         form (sexpr libspec-expr)]
     (cond (prefix-spec? form)
-          (mapcat (fn [f]
-                    (normalize-libspec ctx
-                                       (symbol (str (when prefix (str prefix "."))
-                                                    (first form)))
-                                       f))
-                  (rest children))
+          (do (when prefix
+                (findings/reg-finding! ctx (node->line (:filename ctx)
+                                                       libspec-expr
+                                                       :error
+                                                       :syntax
+                                                       "nested prefix lists are not allowed.")))
```

Comment:
Not sure if this is a helpful error message. Trying to think of a better one.

Reply:
If you are using punctuation, can you also start the sentence with an uppercase? What is the case you are finding here?

Reply:
In general I try to use the same message as the Clojure compiler itself does.

Reply:
This is the case of having nested prefix forms, which doesn't seem to be allowed. Ex:
```
(ns quux
    (:require [clj-kondo.impl
                       [analyzer
                         [foo]
                         [bar]]]))
```
This results in a spec failure, the compiler doesn't seem to have a custom error message for it.
```
Syntax error macroexpanding clojure.core/ns at (REPL:1:1).
((:require [foo [bar [baz :as b]] [baz :as c]])) - failed: Extra input spec: :clojure.core.specs.alpha/ns-form
```

Reply:
So the issue is that nested libspecs actually only support one nesting?

Reply:
@borkdude yes, that's right.

Reply:
> Prefix Lists

> It's common for Clojure code to depend on several libs whose names have
the same prefix. When specifying libs, prefix lists can be used to reduce
repetition. A prefix list contains the shared prefix followed by libspecs
with the shared prefix removed from the lib names. After removing the
prefix, the names that remain must not contain any periods.

A nicer message may be "Prefix lists can only have two levels." perhaps?

Reply:
Sounds good to me, I've changed it.